sympy/physics/vector/printing.py

```diff
@@ -149,7 +149,7 @@ def _print_Derivative(self, der_expr):
         elif dots == 4:
             base = r"\ddddot{%s}" % base
         else: # Fallback to standard printing
-            return LatexPrinter().doprint(der_expr)
+            return super()._print_Derivative(der_expr)
```

Comment:
This change means that
```python
r'\frac{d^{5}}{d t^{5}} \omega{\left(t \right)}\mathbf{\hat{n}_x}'
```
is instead printed as
```python
r'\frac{d^{5}}{d t^{5}} \omega\mathbf{\hat{n}_x}'
```
I assume this is fine?

Reply:
What is the input? Why does it change? Is it because of inheriting the dot notation or something?

Reply:
The input is `dynamicsymbol('omega').diff(t, 5)` or similar.

It changes with this patch because switching to the `LatexPrinter` caused all of the vector customizations to be forgotten.

The question is whether this is deliberate or accidental.

Reply:
What customisation causes the `(t)` to disappear?

Reply:
The `VectorLatexPrinter` customizes printing of `dynamicsymbol` (aka functions of t)

Reply:
Then I guess it's intentional...

Reply:
Sorry, to be clear - do you think the _old_ behavior of short-cutting the customizations was intentional, or the new behavior in this patch was what was originally intended?

Reply:
I don't know the original intention. It comes from #15849

Reply:
What I meant was that the general intention is for those settings to omit the `(t)` so it's not just a bug.

I think it's fine to change that. Someone who doesn't want that should not use those settings.

Reply:
Updated the tests to pass, hopefullly. Realized that almost all of `VectorLatexPrinter._print_Function` can be gutted too, but will deal with that in a future PR.